src/plugins/workspace/server/plugin.ts

```diff
@@ -2,22 +2,32 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-
+import { Observable } from 'rxjs';
```

Comment:
```suggestion

import { Observable } from 'rxjs';
```

One empty line under license header.

Reply:
Fixed.